src/push/index.nativescript.ts

```diff
@@ -0,0 +1,69 @@
+import { messaging, Message } from 'nativescript-plugin-firebase/messaging';
+import { device } from 'tns-core-modules/platform';
+import { formatKinveyUrl } from '../http/utils';
+import { KinveyRequest, RequestMethod } from '../http/request';
+import { Auth } from '../http/auth';
+import { get as getConfig } from '../kinvey/config';
+
+const PUSH_NAMESPACE = 'push';
+
+export async function register(callback: (message: Message) => void, options: any = {}) {
+  // Register for push notifications
+  const {
+    // Whether you want this plugin to automatically display the notifications or just notify the callback. Currently used on iOS only. Default true.
+    showNotifications = true,
+    // Whether you want this plugin to always handle the notifications when the app is in foreground. Currently used on iOS only. Default false.
+    showNotificationsWhenInForeground = true
```

Comment:
The comment says default is `false`, but the code has it set to `true`.

Reply:
The default value for the plugin is false but I think we should change it to default to true for our SDK. I left the comment to read the default value for the plugin. Maybe I should reword the comment to be less confusing.

Reply:
This has been fixed.